src/elements/content-preview/ContentPreview.js

```diff
@@ -129,13 +131,8 @@ type PreviewMetrics = {
     value: number,
 };
 
-type PreviewError = {
-    error: {
-        code: string,
-        details: Object,
-        displayMessage: string,
-        message: string,
-    },
+type PreviewLibraryError = {
+    error: ErrorType,
 };
 
 const InvalidIdError = new Error('Invalid id for Preview!');
```

Comment:
Does this get shown to the user, if so should it be a formatted message?

Reply:
No it doesn't